.gitlab-ci.yml

```diff
@@ -319,6 +319,20 @@ check_warnings:
       fi
   allow_failure:                   true
 
+check_polkadot:
+  stage:                           build
```

Comment:
```suggestion
  stage:                           test
```

Otherwise it is not run while testing a pr.

Reply:
Why not? Does the stage really matter? I just want it to run after the checks are green as this might be a bit heavy.